bootstrap/manifest_template.json

```diff
@@ -125,12 +125,72 @@
     }
   },
   {
+    "id": "redis",
+    "app": {
+      "name": "redis",
+      "meta": {"flynn-system-app": "true"}
+    },
+    "action": "run-app",
+    "release": {
+      "env": {
+        "FLYNN_REDIS": "redis"
+      },
+      "processes": {
+        "redis": {
+          "ports": [{"port": 5432, "proto": "tcp"}],
+          "data": true,
+          "cmd": ["redis"],
+          "service": "redis",
+          "env": {
+            "SINGLETON": "{{ .Singleton }}"
+          },
+          "resurrect": true
```

Comment:
I don't think Redis needs to be ressurected. The once the `controller` has been successfully resurrected it will sync the Redis formations with the state of the world and fix everything up.